test/e2e/main_test.go

```diff
@@ -307,6 +306,71 @@ func genericTestForPersistenceVolume(clusterName string, modifier func(*ispnv1.I
 	verifier(spec)
 }
 
+func TestCheckDataSurviveToShutdown(t *testing.T) {
+	name := "test-data-shutdown-cache"
+	usr := "developer"
+	cacheName := "test"
+	template := "<infinispan>" +
```

Comment:
golang has string literals that make this more palatable, see [here](https://github.com/infinispan/infinispan-operator/pull/210/files#diff-9b3d5baa2e1ef758d21388446f725c84R456).